installation.md

```diff
@@ -69,45 +69,47 @@ should work with any of them, though feature set and interface may vary.
 Here are some plugins we know work well with clangd:
 
 <details>
-<summary markdown="span">Vim</summary>
```

Comment:
I think we should move YCM below a ruler (`---`) rather than removing it entirely.
They support clangd pretty well and there's quite a lot of users there we should doc the setup for.